smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/providers/connectors/ExecutionHolder.java

```diff
@@ -19,6 +22,9 @@
 @ApplicationScoped
 public class ExecutionHolder {
 
+    private static final String REACTIVE_MESSAGING_VERTX_CDI_QUALIFIER = "mp.messaging.connector.vertx.cdi.qualifier";
+    private static final String REACTIVE_MESSAGING_VERTX_CDI_QUALIFIER_REQUIRED = "mp.messaging.connector.vertx.cdi.qualifier.required";
```

Comment:
Do we really need a second property ? 
Can't we just look for "mp.messaging.connector.vertx.identifier" and if it is defined look up the instance with it?

Reply:
+1, just the identifier value should be fine.

Reply:
Maybe not using Identifier, I will test to confirm that. The case it tries to cover is when there is a vertx instance exposed with a cdi qualifier(`Named("vertx")`) but the `Instance<Vertx> ist` in the ExecutionHolder, the `ist.isUnsatisfied()` is `false`, which forces sr rm to inject the instance.

BTW, I will be on holidays this week, will report back next week, thanks :)

Reply:
Done by using only the first option and the `@io.smallrye.common.annotation.Identifier` qualifier only.

The vertx bean from the vertx subsystem is exposed with 2 qualifiers: `@Any` and `@Identifier`, so to select the instance only when the cdi qualifier is specified, I split the `@Default` out in the `else` check block.